charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if semverCompare ">=1.14" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Should there be a check for <=1.13 and have it fail with some useful error?

Reply:
this is not needed in the doc we are mentioning we support kube 1.13+